podium/talks/templates/talks/sessions.html

```diff
@@ -1,35 +1,57 @@
 {% extends "base.html" %}
-{% block content%}
-{%for session in sessions%}
-<div >
-	
-	  <div><h3> Thurs. {{session.date}}</h3></div>
-	  <div >
-	    <h4>{{session.description}}</h4>
-	  </div>
 
-	  <!-- Table -->
-	  <table>
-	  <thead>
-		  <tr>
-			    <th>Title</th>
-			    <th>Speaker</th>
-			    <th>Email</th>
-		    </tr>
-	    </thead>
+{% block content %}
+<div class="container">
+		<div class="row ">
+			<div class="
+				col-xs-12 
+				col-sm-10 col-sm-push-1
+				col-md-8 col-md-push-2
+				col-lg-8 col-lg-push-2
+				" > 
+				<h2 class="bigger">Sessions</h2>
 
-	    <tbody>
+				{%for s in sessions%}
+				<div class="panel panel-default shadow ">
+					  <!-- Default panel contents -->
+					  <div class="panel-heading"><h3> Thurs. {{s.date}}</h3></div>
+					  <div class="panel-body">
+					    <h4>{{s.description}}</h4>
+					  </div>
 
-	     	{%for talk in session.talks_available.all%}
-	    	<tr>
-	    		<td>{{talk.title}}</td>
-	    		<td>{{talk.speaker_name}}</td>
-	    		<td>{{talk.speaker_email}}</td>
-	    	</tr>
-	    	{%endfor%}
+					  <!-- Table -->
+					  <table class="table">
+					  <thead>
+						  <tr>
+							    <th>Title</th>
+							    <th>Speaker</th>
+							    <th>Email</th>
+							    {%if user.is_staff%}	
+							    <th>Status</th>
+							    {%endif%}
+						    </tr>
+					    </thead>
 
-	    </tbody>
-	  </table>
-</div>
-{%endfor%}
-{%endblock%}
\ No newline at end of file
+					    <tbody>
+
+					     	{%for t in s.talks_available.all%}
+					    	<tr>
+					    		<td>{{t.title}}</td>
+					    		<td>{{t.speaker_name}}</td>
+					    		<td>{{t.speaker_email}}</td>
```

Comment:
We shouldn't be publicly exposing the speaker's email address without their permission.  This can be an option in the future if we add an "opt-in" flag of some sort, but in the current version this just invites spam bots to scrape our site for email addresses.